web/handlers.go

```diff
@@ -1,3 +1,156 @@
 // Package web provides managing HTTP protocol.
 // Implement CRUD operations for filters over HTTP API.
 package web
+
+import (
+	"encoding/json"
+	"net/http"
+	"strconv"
+
+	"github.com/google/uuid"
+	"github.com/gorilla/mux"
+	"github.com/lvl484/positioning-filter/repository"
+)
+
+const (
+	inputUserID = "user_id"
+	inputName   = "name"
+)
+
+type repo struct {
+	filters repository.Filters
+}
+
+func newRepo(filters repository.Filters) *repo {
+	return &repo{
+		filters: filters,
+	}
+}
+
+func (repo *repo) AddFilter(w http.ResponseWriter, r *http.Request) {
+	vars := mux.Vars(r)
+	userIDstring := vars[inputUserID]
+
+	userid, err := uuid.Parse(userIDstring)
+	if err != nil {
```

Comment:
Missed errors logging

Reply:
fixed

Reply:
You have application logger, you need to use it
https://github.com/lvl484/positioning-filter/blob/ef97f687b9c4fe729417c132c26429435e2eba40/logger/logger.go#L21